sources/packages/web/src/types/contracts/supporting-user/SupportingUserContracts.ts

```diff
@@ -67,3 +67,28 @@ export interface ApplicationSupportingUsersDTO {
   supportingUserId: number;
   supportingUserType: SupportingUserType;
 }
+
+export interface AddressInfo {
+  addressLine1: string;
+  addressLine2?: string;
+  province: string;
+  country: string;
+  city: string;
+  postalCode: string;
+}
+export interface ContactInfos {
```

Comment:
Just wondering why the interface name has taken s at the end? it can be ContactInfo IMO.

Reply:
in `web` there is already another interface with the same name. do you have any better name suggestion

Reply:
If it is specific for supporting user, then SupportingUserContactInfo IMO.

Reply:
Is the address info shared in API also?
If it is shared in the API I would say that we can share in Vue also moving it under _types_.

Reply:
yes, in API `ContactInfo` is shared between supporting users and students, actually, its inside `web/scr/type/`, did u mean to add to a shared file? ya i will